tvb_framework/tvb/interfaces/web/controllers/project/project_controller.py

```diff
@@ -733,3 +735,5 @@ class EditForm(formencode.Schema):
     administrator = validators.UnicodeString(not_empty=False)
     project_id = validators.UnicodeString(not_empty=False)
     visited_pages = validators.UnicodeString(not_empty=False)
+    disable_imports = validators.Bool()
+    max_operation_size = validators.UnicodeString(not_empty=False)
```

Comment:
Maybe here we can use a number validator class for `max_operation_size` (e.g. `validators.Int(...`) ?